mui/demos/showcase/src/index.tsx

```diff
@@ -2,19 +2,34 @@ import 'react-app-polyfill/ie11';
 import 'react-app-polyfill/stable';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { CssBaseline, ThemeProvider, createMuiTheme } from '@material-ui/core';
+import {
+    CssBaseline,
+    ThemeProvider,
+    StyledEngineProvider,
+    createTheme,
+    adaptV4Theme,
+} from '@mui/material';
+import { Theme } from '@mui/material/styles';
+import '@mui/styles';
 import * as BLUIThemes from '@brightlayer-ui/react-themes';
 import '@brightlayer-ui/react-themes/open-sans';
 import { App } from './App';
 import reportWebVitals from './reportWebVitals';
 import './index.css';
 
+declare module '@mui/styles/defaultTheme' {
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface DefaultTheme extends Theme {}
+}
+
 ReactDOM.render(
     <React.StrictMode>
-        <ThemeProvider theme={createMuiTheme(BLUIThemes.blue)}>
-            <CssBaseline />
-            <App />
-        </ThemeProvider>
+        <StyledEngineProvider injectFirst>
+            <ThemeProvider theme={createTheme(adaptV4Theme(BLUIThemes.blue))}>
```

Comment:
When our theme is converted over to MUI5, this `adaptV4Theme` wrapper will go away right?